game/plugins/src/main/kotlin/gg/rsmod/plugins/content/mechanics/prayer/prayers.plugin.kts

```diff
@@ -1,5 +1,30 @@
 package gg.rsmod.plugins.content.mechanics.prayer
 
+/**
+ * Checks if Retribution is active and deals
+ * ((prayer lvl)/4) damage to nearby enemies
+ */
+on_player_pre_death {
+    if (Prayers.isActive(player, Prayer.RETRIBUTION)) {
+        val targets = player.getNeighbourPawns()
+        for (target in targets) {
+            if (!target.isDead()) {
```

Comment:
Would rater see something like `if (target.isDead()) { continue; }`. 
this probably makes it a bit more readable and prevents from stacking indentions.